data/language/dutch.txt

```diff
@@ -1078,20 +1078,20 @@ STR_1076    :Winkelstand
 STR_1077    :Draaistand
 STR_1078    :Voorwaartse rotatie
 STR_1079    :Achterwaartse rotatie
-STR_1080    :Film: {ENDQUOTES}Avenging aviators{ENDQUOTES}
-STR_1081    :3D-film: {ENDQUOTES}Mouse tails{ENDQUOTES}
+STR_1080    :Film: {ENDQUOTES}Avenging Aviators{ENDQUOTES}
+STR_1081    :3D-film: {ENDQUOTES}Mouse Tails{ENDQUOTES}
 STR_1082    :Ruimteringenmodus
 STR_1083    :Beginnersstand
-STR_1084    :LIM-lancering
-STR_1085    :Film: {ENDQUOTES}Thrill riders{ENDQUOTES}
-STR_1086    :3D-film: {ENDQUOTES}Storm chasers{ENDQUOTES}
-STR_1087    :3D-film: {ENDQUOTES}Space raiders{ENDQUOTES}
+STR_1084    :Lineaire inductiemotor-lancering
```

Comment:
'Lineaire inductiemotor' is wel correct, maar het moet 'Lineaireinductiemotorlancering' zijn, al dan niet met verduidelijkende streepjes. Bijvoeglijke naamwoorden moeten aan elkaar in een samenstelling als het over een gedeelte gaat en niet het geheel. Zie [hier](http://www.beterspellen.nl/website/index.php?pag=91). Ik zou zeggen dat 'LIM' net zo bekend is als 'lineaire inductiemotor', zeker onder achtbaan- en/of natuurkundekenners.